File: ItCameDroid-AS/app/src/main/java/br/DroidLib/Vec2.java
package br.DroidLib;

public class Vec2 {
    public final float x;
    public final float y;

    public Vec2(float i, float j) {
        this.x = i;
        this.y = j;
    }

    public boolean isCloseEnoughToConsiderEqualTo(Vec2 position) {
        return false;
    }
}

File: app/src/main/java/br/odb/giovanni/engine/Vec2.kt
package br.odb.giovanni.engine

import kotlin.math.abs

class Vec2(var x: Float = 0.0f, var y: Float = 0.0f) {

	fun isCloseEnoughToConsiderEqualTo(position: Vec2): Boolean {
		return abs(x - position.x) < 1.0f && abs(y - position.y) < 1.0f
	}
}